EventFilter/Utilities/src/FedRawDataInputSource.cc

```diff
@@ -532,7 +532,7 @@ inline evf::EvFDaqDirector::FileStatus FedRawDataInputSource::getNextEvent() {
       chunkIsFree_ = true;
     } else {
       //header was contiguous, but check if payload fits the chunk
-      if (eventChunkSize_ - currentFile_->chunkPosition_ < msgSize) {
+      if (eventChunkSize_ - (uint32_t)currentFile_->chunkPosition_ < msgSize) {
```

Comment:
won't this make `currentFile_->chunkPosition_` wrap around when it is larger then 4 GB ?